dataproxy/vercel-cli-serverless-functions/run.sh

```diff
@@ -35,8 +35,8 @@ else
   exit 1
 fi
 
-# Check the Vercel Build Logs for "| dataproxy" in generate
-if echo "${OUTPUT}" | grep -q '| dataproxy'; then
+# Check the Vercel Build Logs for "engine=none" in generate
+if echo "${OUTPUT}" | grep -q 'engine=none'; then
```

Comment:
New output in `generate`?

(Answer as comment, not emoji reaction please so I get notification. Thanks.)

Reply:
Indeed.